### DISCUSSION TITLE
enh: support openai chat param metadata

### DISCUSSION THREAD

**Discussion by bdqfork** [open-webui/open-webui#19356] [Ideas]
# Feature Request

**Important: Before submitting a feature request, please check whether a similar issue or feature request has already been posted in the Issues or Discussions section. It's likely we're already tracking it. In case of uncertainty, initiate a discussion post first. This helps us all to efficiently focus on improving the project.**

**Let's collaborate respectfully. If you bring negativity, please understand our capacity to engage may be limited. If you're open to learning and communicating constructively, we're more than happy to assist you. Remember, Open WebUI is a volunteer-driven project maintained by a single maintainer, supported by our amazing contributors who also manage full-time jobs. We respect your time; please respect ours. If you have an issue, We highly encourage you to submit a pull request or to fork the project. We actively work to prevent contributor burnout to preserve the quality and continuity of Open WebUI.**

**Is your feature request related to a problem? Please describe.**
I implemented a workflow using langgraph, utilizing the persistent memory provided by langgraph. I hope to be able to perceive the chat-id as the key for the memory.

**Describe the solution you'd like**
![image](https://github.com/user-attachments/assets/95532efa-4584-438b-a0a4-4bae1f8bd62d)
We can support the metadata parameter, allowing the input of parameters like chat-id. For OpenAI users, this enables tracking calls coming from open-webui. On the other hand, it also allows for obtaining relevant information in the pipeline, achieving more flexible expansion.

**Describe alternatives you've considered**

**Additional context**

**Comment by larrykoo711** [comment]
+1 we need this. metadata is great extentions params.
